phone/src/os/call/hooks/useCall.ts

```diff
@@ -21,11 +22,18 @@ interface CallHook {
 
 export const useCall = (): CallHook => {
   const [call, setCall] = useCurrentCall();
-  // const [dialRing, setDialRing] = useState(false);
   const myPhoneNumber = useMyPhoneNumber();
   const [t] = useTranslation();
   const { addAlert } = useSnackbar();
-  // const { endDialTone, startDialTone } = useDialingSound();
+  const { endDialTone, startDialTone } = useDialingSound();
+
+  useEffect(() => {
+    if (call?.isTransmitter && !call.is_accepted) {
```

Comment:
Can't `is_accepted` also be null here too?

Reply:
Yes but this conditional should handle for that, does it not?. I do think that our call state API needs a more declarative interface though?

Reply:
Yes, the interface has to be improved. We can just keep it as it is. This is going to develop so, we can test there.